gio/src/list_model.rs

```diff
@@ -19,15 +30,86 @@ impl<T: IsA<ListModel>> ListModelExtManual for T {
         }
```

Comment:
You can implement ^---- this one via `self.iter().collect::<Result<Vec<T>, _>>>().unwrap()` now

Reply:
I'd keep `snapshot` as it is, because now it's faster and the implementation is really small anyway